### PR TITLE
W-064

### PR DESCRIPTION
**Pull request by szymon-muszynski**
User profile template

### CODE REVIEW

File: frontend/src/components/user-profile/setCanvasPreview.jsx
Comment (WOJTASK1) [inline frontend/src/components/user-profile/setCanvasPreview.jsx:1]: please delete comments

File: frontend/src/components/user-profile/setCanvasPreview.jsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
+const setCanvasPreview = (
```
Comment (wojciechwelfle) [inline frontend/src/components/user-profile/setCanvasPreview.jsx:1]: delete comments

File: frontend/src/components/user-profile/CloseIcon.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+const CloseIcon = () => (
```
Comment (wojciechwelfle) [inline frontend/src/components/user-profile/CloseIcon.jsx:1]: move to folder icons

File: frontend/src/components/user-profile/PencilIcon.jsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+const PencilIcon = () => (
```
Comment (wojciechwelfle) [inline frontend/src/components/user-profile/PencilIcon.jsx:1]: move to icon folder

File: frontend/src/components/user-profile/ProfileCard.css [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+.avatar-container {
+  position: relative;
+}
+
+.avatar {
+  width: 200px;
+  height: 200px;
+  border-radius: 50%;
+  border: 2px solid #ccc;
```
Comment (wojciechwelfle) [inline frontend/src/components/user-profile/ProfileCard.css:16]: global colors

File: frontend/src/components/user-profile/UserProfile.css [hunk 5]
```diff
@@ -0,0 +1,109 @@
+.profile-wrapper {
+    display: flex;
+    flex-direction: column;
+    padding: 5px;
+    gap: 20px;
+    margin-top: 100px;
+    align-items: flex-start;
+    width: 80%;
+    margin-left: auto;
```
Comment (wojciechwelfle) [inline frontend/src/components/user-profile/UserProfile.css:9]: margin: auto

File: frontend/src/components/user-profile/UserProfile.css [hunk 6]
```diff
@@ -0,0 +1,109 @@
+.profile-wrapper {
+    display: flex;
+    flex-direction: column;
+    padding: 5px;
+    gap: 20px;
+    margin-top: 100px;
```
Comment (wojciechwelfle) [inline frontend/src/components/user-profile/UserProfile.css:6]: delete margin top and gap

File: frontend/src/components/user-profile/UserProfile.css [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+}
+
+.picture-column, .textbox-column {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.profile-image {
+    border: 2px solid black;
```
Comment (wojciechwelfle) [inline frontend/src/components/user-profile/UserProfile.css:31]: global colors

File: frontend/src/components/user-profile/UserProfile.css [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+        color: var(--text-color);
+    }
+
+    .textbox-column {
+        width: 100%;
+        height: auto;
+        padding: 10px;
+        background-color: var(--background-color);
+        color: var(--text-color);
+        max-height: 1000px;
```
Comment (wojciechwelfle) [inline frontend/src/components/user-profile/UserProfile.css:76]: 1000px?

**Review by wojciechwelfle — CHANGES_REQUESTED** [review]
change css, delete comments

**Review by wojciechwelfle — APPROVED** [review]
lgtm

**Pull request merged**